bootstrap_actions/status_metrics.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+(
+    INSTANCE_ROLE=$(jq .instanceRole /mnt/var/lib/info/extraInstanceData.json)
+
+    #only log on master to avoid duplication
+    if [[ "$INSTANCE_ROLE" != '"MASTER"' ]]; then
+        exit 0
+    fi
+
+    source /opt/emr/logging.sh
+
+    function log_wrapper_message() {
+        log_aws_clive_message "$${1}" "status_metrics.sh" "$${PID}" "$${@:2}" "Running as: ,$USER"
+    }
+
+  log_wrapper_message "Start running status_metrics.sh Shell"
+
+  STEP_DETAILS_DIR=/mnt/var/lib/info/steps
+  CORRELATION_ID_FILE=/opt/emr/correlation_id.txt
+  SNAPSHOT_TYPE_FILE=/opt/emr/snapshot_type.txt
+  EXPORT_DATE_FILE=/opt/emr/export_date.txt
+
+  DATE=$(date '+%Y-%m-%d')
+  CLUSTER_ID=$(jq '.jobFlowId' < /mnt/var/lib/info/job-flow.json)
+  CLUSTER_ID="$${CLUSTER_ID//\"}"
+
+  FAILED_STATUS="FAILED"
+  COMPLETED_STATUS="COMPLETED"
+  CANCELLED_STATUS="CANCELLED"
+  FINAL_STEP_NAME="run-clive"
```

Comment:
this final step name is now hard coded in more than one file. It is also used in the update dynamo script. It might be time to make it a local variable in our local.tf so we only need to alter it in one place